src/components/Hero/Hero.module.css

```diff
@@ -41,7 +41,7 @@
 }
 
 .location-pin {
-	width: 1rem;
+	width: 9.5%;
```

Comment:
wouldn't the solution be to use the `object-fit` CSS property?
This percentage seems like a magic number